src/intro.js

```diff
@@ -0,0 +1,12 @@
+(function (factory) {
+   if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define(['moment'], factory);
+    } else if ( typeof exports === 'object' ) {
+        // Node/CommonJS
+        module.exports = factory();
```

Comment:
Shouldn't you be doing `factory(require('moment'))` here?  The moment-less build wont have moment bundled in, so moment will be undefined.